### ISSUE TITLE
Please add support for Laravel 6

### ISSUE THREAD

**Issue by Kaishiyoku** [austinheap/laravel-database-encryption#54]

**Comment by roelofr** [comment]
See #53, it's basically just waiting for the maintainer to press "merge".

**Comment by SuryaWebfox** [comment]
Yes, still waiting. And I need support for v7. Not feeling hopeful here :(

**Comment by roelofr** [comment]
> Yes, still waiting. And I need support for v7. Not feeling hopeful here :(

Why for Laravel 7? You're much better off using custom casts for that (like I did [here][1])

Maybe I should make a simple library of this too

[1]: https://github.com/gumbo-millennium/website/tree/4a9e8558719c44c3616666afef49912fb3ce0621/app/Casts

**Comment by RandyBooth** [comment]
https://github.com/betterapp/laravel-db-encrypter

**Comment by SuryaWebfox** [comment]
Thanks for the tip, Randy

**Comment by roelofr** [comment]
> > Yes, still waiting. And I need support for v7. Not feeling hopeful here :(
> 
> Why for Laravel 7? You're much better off using custom casts for that (like I did [here](https://github.com/gumbo-millennium/website/tree/4a9e8558719c44c3616666afef49912fb3ce0621/app/Casts))
> 
> Maybe I should make a simple library of this too

I made this a simple library, for those using Laravel 7.x and newer.

It's available [here][1], and has a compat method to seamlessly switch over from this package, unless you changed the default data too much.

[1]: https://github.com/roelofr/laravel-encryption-cast

**Comment by heyaj19** [comment]
@roelofr does this decrypt when you read from the DB? Because its not doing so when using tinker

**Comment by roelofr** [comment]
> @roelofr does this decrypt when you read from the DB? Because its not doing so when using tinker

If you're talking about the Laravel 7 cast I made, please open an issue there and don't go off-topic here.

**Comment by AndreasTsika** [comment]
https://github.com/ilyakonovaloff/laravel-database-encryption
This one continues the same package if you don't want to do it locally.
But if you use that remember to put on composer.json this package too:  https://github.com/laravel/helpers
Because the one above didn't change the Str_ that is described here: https://laravel.com/docs/6.x/upgrade#helpers

**Comment by TsikaAndreas** [comment]
https://github.com/ilyakonovaloff/laravel-database-encryption
This one continues the same package if you don't want to do it locally.
But if you use that remember to put on composer.json this package too:  https://github.com/laravel/helpers
Because the one above didn't change the Str_ that is described here: https://laravel.com/docs/6.x/upgrade#helpers
